packages/carbon-graphs/README.md

```diff
@@ -1,20 +1,112 @@
 # Carbon Graphs
 
 [![Build Status](https://travis-ci.com/cerner/terra-graphs.svg?branch=main)](https://travis-ci.com/cerner/terra-graphs)
+![npm](https://img.shields.io/npm/v/@cerner/carbon-graphs)
 
-A graphing library built using d3 based on Cerner design standards.
+A vanilla JavaScript graphing library built using d3 based on Cerner design standards.
 
+<br>
+
+
+- [Installing Carbon](#installing-carbon)
 - [Getting Started](#getting-started)
-- [Documentation](../terra-graphs/docs/README.md)
+- [Graph Types](#graph-types)
 - [LICENSE](#license)
 
+## Installing Carbon
+
+- Install from [npmjs](https://www.npmjs.com/package/@cerner/carbon-graphs): 
```

Comment:
nit: Install from [npmjs](https://www.npmjs.com/package/@cerner/carbon-graphs) using the below command

Reply:
Updated here: 9515cb45a727e543c1979448ea6c6255ec8d7328